bciers/apps/reporting/src/app/components/facility/FacilityEmissionAllocationPage.tsx

```diff
@@ -11,7 +11,7 @@ export default async function FacilityEmissionAllocationPage({
 }) {
   const orderedActivities = await getOrderedActivities(version_id, facility_id);
   const initialData = await getEmissionAllocations(version_id, facility_id);
-
+  console.log(initialData);
```

Comment:
left over debugging line

Reply:
Oops, I just went ahead and merged it immediately after approval. My bad. I'll just take this out in my current PR